src/Controller/ContentPreviewController.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_translation\Controller;
+
+use Drupal\tmgmt\JobItemInterface;
+use Drupal\tmgmt_content\Controller\ContentTranslationPreviewController;
+
+/**
+ * Extends the original TMGMT preview controller.
+ */
+class ContentPreviewController extends ContentTranslationPreviewController {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function title(JobItemInterface $tmgmt_job_item) {
```

Comment:
It would be nice to add a comment like: "Show the translated title on the translation preview page."